handler/handler.ts

```diff
@@ -1,3 +1,4 @@
+require('dotenv').config({ path:'../.env' })
```

Comment:
same here: shouldn't this be .env.dev?

If you did intend to use .env instead of .env.dev, then we should also make that consistent in the dockerfile